packages/leancode_lint/README.md

```diff
@@ -6,11 +6,19 @@ Lint rules used internally in LeanCode projects.
 
 ## Usage
 
-Add `leancode_lint` as a normal dependency to access the `unawaited` function when using dart version lower than 2.14.
+Add `leancode_lint` as a dev dependency.
+
+```yaml
```

Comment:
I'd put this snippet and the paragraph below under a h3 `Dart <2.14`

Reply:
done